src/api/public/apidocs-new/OBS-v2.10.50.yaml

```diff
@@ -231,6 +231,12 @@ paths:
     $ref: 'paths/search_request.yaml'
   /search/request/id:
     $ref: 'paths/search_request_id.yaml'
+  /search/channel:
+    $ref: 'paths/search_channel.yaml'
+  /search/channel/binary:
+    $ref: 'paths/search_channel_binary.yaml'
+  /search/channel/binary/id:
+    $ref: 'paths/search_channel_binary_id.yaml'
```

Comment:
We should keep the endpoints alphabetically orderer.These should go between the `/search` and `/search/issue` endpoints.

Reply:
done